DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Centre/Reports/ReportsViewModel.cs

```diff
@@ -29,7 +29,7 @@ public class UsageStatsTableRow
     {
         public UsageStatsTableRow(MonthOfActivity monthOfActivity)
         {
-            Period = DateTime.Parse($"{monthOfActivity.Year}-{monthOfActivity.Month}-01").ToString("MMMM, yyyy");
+            Period = DateTime.Parse($"{monthOfActivity.Year}-{monthOfActivity.Month}-01").ToString("MMMM yyyy");

```

Comment:
This format (no comma) looks better in the graph labels and in my view on the table, but looking at it now I think it probably should be a parameter of the view model so that it can be set to whatever is required.

Reply:
Agreed, looks better without comma, but it seems sensible to have it as an argument.

Reply:
We can wait until adding the filters, because we will need to modify the format then anyway 👍